spec/section_spec.rb

```diff
@@ -53,9 +53,11 @@ class PageWithSection < SitePrism::Page
       subject(:section) { Page.section(:incorrect_section, '.section') }
 
       it 'should raise an ArgumentError' do
-        class Page < SitePrism::Page
-        end
-        expect { section }.to raise_error ArgumentError, 'You should provide section class either as a block, or as the second argument'
+        class Page < SitePrism::Page; end
+
+        expect { section }
+          .to raise_error(ArgumentError)
+          .with_message('You should provide section class either as a block, or as the second argument')
```

Comment:
trivial: should error messages end in a period?

Reply:
They should. Not sure why this passes. Will re-triage